spark/src/main/scala/org/apache/iceberg/spark/SparkTableUtil.scala

```diff
@@ -19,18 +19,22 @@
 
 package org.apache.iceberg.spark
 
+import com.google.common.collect.ImmutableMap
 import com.google.common.collect.Maps
 import java.nio.ByteBuffer
 import java.util
+import java.util.UUID
 import org.apache.hadoop.conf.Configuration
 import org.apache.hadoop.fs.{Path, PathFilter}
-import org.apache.iceberg.{DataFile, DataFiles, Metrics, MetricsConfig, PartitionSpec}
-import org.apache.iceberg.hadoop.HadoopInputFile
+import org.apache.iceberg._
```

Comment:
Please don't use wildcard imports.

Reply:
hmm... The Idea IDE always optimizes the imports when adding a new one from Alt+Enter. Let me update it.

Reply:
It looks like we need to import 9 entities which exceed 100 line length limit. From spark's coding style, when importing more than 6 entities it prefers to use a wildcard. Just want to confirm which one do we prefer? line break or following spark coding style for scala?

Reply:
We never use wildcard imports.

Reply:
I see, will update in next commit.